README.md

```diff
@@ -160,6 +160,19 @@ Vault instance before starting Vault:
   - `/var/vcap/jobs/vault/tls/other_tls_cert/cert.pem`
   - `/var/vcap/jobs/vault/tls/other_tls_cert/key.pem`
 
+**WARNING** Prior to 1.0.0 release, the `VAULT_SKIP_VERIFY` environment variable is set
```

Comment:
This note summarizes a pretty cool feature/fix. I think it should have its own section. Something like

```
### Monit Script Configuration

In order to enable features like zero downtime redeploys this Bosh release 
bundles scripts that utilize the Vault CLI. Manifest properties are available to 
explicitly set the value of the `VAULT_SKIP_VERIFY` and `VAULT_ADDR` 
environment variables in the context of these monit scripts:

  properties:
    vault.tls_skip_verify: false
    vault.addr: "https://127.0.0.1:8200"

**WARNING** Prior to the 1.0.0 release the `VAULT_SKIP_VERIFY` 
environment variable was set if the Vault address contained...
```

Reply:
Will take care of it.